compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -347,52 +351,61 @@ void addPadPackBasedPassPipeline(OpPassManager &pm,
     AMDAIEBufferizeToAllocationOptions bufferizeOptions2;
     bufferizeOptions2.memorySpace = 2;
     bufferizeOptions2.bufferizeOperand = BufferizeOperand::Input;
-    modulePassManager.addNestedPass<func::FuncOp>(
+    modulePassManager.addPass(
         createAMDAIEBufferizeToAllocationPass(bufferizeOptions2));
   }
-
+  /*
   // Lower to UKernels
   {
     AMDAIELowerToUKernelsOptions options;
     options.passPipeline = AIEPassPipeline::PadPackPipeline;
     options.pathToUkernels = clPathToUkernels;
-    modulePassManager.addNestedPass<func::FuncOp>(
+    modulePassManager.addPass(
```

Comment:
Make this pass a `FunctionOpInterface` pass, it can actually be an unspecified pass.